concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/05 Hide columns.md

```diff
@@ -0,0 +1,103 @@
+TreeList can hide columns to adapt to screens of different sizes. To add this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) to **true**.
+
+Each column has a unique hiding priority. The rightmost column is the base column with the priority of 0. This value is incremented by 1 for columns from right to left; column with the lowest priority is hidden first.
+
+You can use the **columns[]**.[hidingPriority](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#hidingPriority) property to override the default behavior: specify custom hiding priorities for those columns only that you want to hide. In this case, the **columnHidingEnabled** property can be omited.
```

Comment:
- "the default behavior" - what was the default behavior? Is it the one described after the colon? Or the one from the previous paragraph?
- "for those columns only that you want to hide" - mind the word order.
- "omited" - it's "omitted" with double "t".